src/applications/static-pages/medical-copays-cta/components/App/index.js

```diff
@@ -24,10 +28,8 @@ export const App = ({ loggedIn, show, toggleLoginModal }) => {
         <p>With this tool, you can:</p>
       ) : (
         <p>
-          Try signing in with your <strong>DS Logon</strong>,{' '}
-          <strong>My HealtheVet</strong>, or <strong>ID.me</strong> account. If
-          you don’t have any of those accounts, you can create one now. When you
-          sign in or create an account, you’ll be able to:
+          Sign in with your existing <ServiceProvidersText isBold /> account.{' '}
```

Comment:
CC @RLHecht @DanielleThierryUSDSVA

Reply:
We should use contractions in the sentence: If you don't have one of these accounts, you'll be able to create a free...

Reply:
Actually: If you don't have one of these account, you can create a free... (change **you'll be able to** to **can**